### DISCUSSION TITLE
Se me recarga la pagina  de Teams

### DISCUSSION THREAD

**Discussion by gladys5** [Launch-X-Latam/MisionFrontEnd#491] [Team Posts]
no puedo entrar a teams, se me recarga hasta el infinito, a alguien mas le pasa?

**Comment by AlvaroFierro** [comment]
Cuando son ese tipo de errores `launch-x` o cualquier otro tipo de organizacion no es que ellos tengan problemas ya que usan un SaaS, en este caso es microsoft.com quien se encarga de ello. 

Cuando es eso y si el error persiste despues de unas horas, borra los datos de navegacion y la cache

**Comment by gladys5** [comment]
gracias,lo haré
